test/e2e/data/infrastructure-docker/cluster-template-machine-pool.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: DockerCluster
+metadata:
+  name: '${ CLUSTER_NAME }'
```

Comment:
```suggestion
  name: '${CLUSTER_NAME}'
```

Reply:
I've seen extra spaces cause issues with envsubst in the past so let's remove all extra spaces to be safe